src/common/RESTBase.py

```diff
@@ -9,34 +9,33 @@ class RESTBase:
     def __init__(self, upperRESTSrvcApp: object, indent_lvl: int) -> None:
         self._upperRESTSrvcApp = upperRESTSrvcApp;
 
-    def asjson(self, data: dict):
+    def asjson(self, data: dict, statuscode: int = 200):
+        cherrypy.response.status = statuscode
         return data
 
-    def asjson_error(self, data):
+    def asjson_error(self, data, statuscode: int = 400):
+        cherrypy.response.status = statuscode
         return {"error": data}
     
-    def asjson_info(self, data):
+    def asjson_info(self, data, statuscode: int = 200):
+        cherrypy.response.status = statuscode
         return {"info": data}
 
     @cherrypy.tools.json_out()
     def GET(self, *path, **args):
-        cherrypy.response.status = 400
-        return self.asjson_error("bad request")
+        return self.asjson_error("bad request", 405)
```

Comment:
In the old version, you had `cherrypy.response.status = 400`.
Why did you change to 405? I think it should be like this:
```suggestion
        return self.asjson_error("bad request", 405)
```

Same is for the other lines of code where you have 405

Reply:
Because I think it's better. 405 stands for "method not allowed" and this is the case, the method is not allowed if not explicitly implemented

Reply:
Ok good, thanks for the explanation! Now I understand why you did this change